p2p/switch.go

```diff
@@ -315,11 +314,15 @@ func (sw *Switch) DialSeeds(addrBook *AddrBook, seeds []string) error {
 		addrBook.Save()
 	}
 
+	// Ensure we have a completely undeterministic PRNG.
+	// See Issue https://github.com/tendermint/tendermint/issues/875.
```

Comment:
Maybe instead of commenting the link to the issue lets just note here that cmn.RandInt64() draws from a seed that's initialized with OS entropy on process start.

Reply:
Nice suggestion. I've literally copied it verbatim #innerMelaniaTrump #amFinnaPlagiarizeThatSuggestion. I'll force push in a second.